sdk/core/core-rest-pipeline/package.json

```diff
@@ -87,7 +87,7 @@
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
     "@azure/core-auth": "^1.3.0",
-    "@azure/core-tracing": "1.0.0-preview.13",
+    "@azure/core-tracing": "^1.0.0",
```

Comment:
Yayy!

Reply:
Does this also mean that opentelemetry has going GA as well?

Reply:
@weshaggard do you mean the `@azure/opentelemetry-instrumentation-azure-sdk` package? That is not going GA, this is just the core interfaces that package plugs into (similar to .NET's ETW handling code)

Reply:
OK I though this package had a direct dependency on it.

Reply:
> OK I though this package had a direct dependency on it.

Not anymore, thankfully. :)